client/my-sites/checkout/composite-checkout/payment-methods/ebanx-tef.js

```diff
@@ -0,0 +1,491 @@
+							obj[ key ] = {
+								value: value.value,
+								isTouched: true,
+							};
+							return obj;
+						}, {} ),
+					};
+			}
+			return state;
+		},
+		actions,
+		selectors,
+	} );
+
+	return { ...store, actions, selectors };
+}
+
+export function createEbanxTefMethod( { store, stripe, stripeConfiguration } ) {
+	return {
+		id: 'brazil-tef',
+		label: <EbanxTefLabel />,
+		activeContent: <EbanxTefFields stripe={ stripe } stripeConfiguration={ stripeConfiguration } />,
+		submitButton: (
+			<EbanxTefPayButton
+				store={ store }
+				stripe={ stripe }
+				stripeConfiguration={ stripeConfiguration }
+			/>
+		),
+		inactiveContent: <EbanxTefSummary />,
+		getAriaLabel: () => 'Transferência bancária',
+	};
+}
+
+function EbanxTefFields() {
+	const { __ } = useI18n();
+
+	const fields = useSelect( ( select ) => select( 'ebanx-tef' ).getFields() );
+	const getField = ( key ) => fields[ key ] || {};
+	const getFieldValue = ( key ) => getField( key ).value ?? '';
+	const getErrorMessagesForField = ( key ) => {
+		const managedValue = getField( key );
+		if ( managedValue?.isRequired && managedValue?.value === '' ) {
+			return [ __( 'This field is required.' ) ];
+		}
+		return managedValue.errors ?? [];
+	};
+	const { setFieldValue } = useDispatch( 'ebanx-tef' );
+
+	const customerName = useSelect( ( select ) => select( 'ebanx-tef' ).getCustomerName() );
+	const customerBank = useSelect( ( select ) => select( 'ebanx-tef' ).getCustomerBank() );
+	const { changeCustomerName, changeCustomerBank } = useDispatch( 'ebanx-tef' );
+	const { formStatus } = useFormStatus();
+	const isDisabled = formStatus !== 'ready';
+	const countriesList = useCountryList( [] );
+
+	return (
+		<EbanxTefFormWrapper>
+			<EbanxTefField
+				id="ebanx-tef-cardholder-name"
+				type="Text"
+				autoComplete="cc-name"
+				label={ __( 'Your name' ) }
+				value={ customerName?.value ?? '' }
+				onChange={ changeCustomerName }
+				isError={ customerName?.isTouched && customerName?.value.length === 0 }
+				errorMessage={ __( 'This field is required' ) }
+				disabled={ isDisabled }
+			/>
+			<BankSelector
+				id="ebanx-tef-bank-selector"
+				value={ customerBank?.value ?? '' }
+				onChange={ changeCustomerBank }
+				label={ __( 'Bank' ) }
+				isError={ customerBank?.isTouched && customerBank?.value.length === 0 }
+				errorMessage={ __( 'This field is required' ) }
+				disabled={ isDisabled }
+			/>
+			<div className="ebanx-tef__contact-fields">
+				<CountrySpecificPaymentFieldsUI
+					countryCode={ 'BR' } // If this payment method is available and the country is not Brazil, we have other problems
+					countriesList={ countriesList }
+					getErrorMessage={ getErrorMessagesForField }
+					getFieldValue={ getFieldValue }
+					handleFieldChange={ setFieldValue }
+					disableFields={ isDisabled }
+				/>
+			</div>
+		</EbanxTefFormWrapper>
+	);
+}
+
+// TODO: Merge this with payment-methods/credit-card/contact-fields.js
```

Comment:
Maybe we should make a new component inside `my-sites/checkout/composite-checkout/` somewhere that includes this wrapper. Like `CompositeCheckoutCountrySpecificPaymentFields` or similar.